WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyRing/PrivacyRingViewModel.cs

```diff
@@ -39,53 +40,59 @@ public PrivacyRingViewModel(WalletViewModel walletViewModel)
 
 		PreviewItems.Add(PrivacyTile);
 
-		_itemsSourceList
+		SetupCancel(enableCancel: true, enableCancelOnEscape: true, enableCancelOnPressed: true);
+	}
+
+	public PrivacyControlTileViewModel PrivacyTile { get; }
+
+	public ObservableCollectionExtended<PrivacyRingItemViewModel> Items { get; } = new();
+	public ObservableCollectionExtended<IPrivacyRingPreviewItem> PreviewItems { get; } = new();
+
+	public Wallet Wallet { get; }
+
+	[System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "Uses DisposeWith()")]
+	protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disposables)
+	{
+		var itemsSourceList = new SourceList<PrivacyRingItemViewModel>();
+
+		itemsSourceList
 			.Connect()
 			.ObserveOn(RxApp.MainThreadScheduler)
 			.Bind(Items)
 			.DisposeMany()
-			.Subscribe();
-
-		_coinsUpdated =
-			walletViewModel.UiTriggers.PrivacyProgressUpdateTrigger
-						  .Merge(walletViewModel
-						  .WhenAnyValue(w => w.IsCoinJoining)
-						  .ToSignal());
+			.Subscribe()
+			.DisposeWith(disposables);
 
-		_coinsUpdated
-			.Select(_ => walletViewModel.Wallet.GetPockets())
-			.ObserveOn(RxApp.MainThreadScheduler)
-			.Subscribe(RefreshCoinsList);
-
-		_disposables.Add(_itemsSourceList);
+		_walletViewModel.UiTriggers
+						.PrivacyProgressUpdateTrigger
+						.Skip(2)
```

Comment:
IMHO, this isn't the way to fix the problem. The real reason why it updates twice is that `UiTriggers` include the usage of Throttle. Skipping the first 2 notifications is brittle.

My suggestions is that you tackle the problem at its source. 
Removing the throttles from UiTriggers and adding them where they make sense.

Reply:
@SuperJMN The thing is that in this specific case I DO want to skip the first 2 because the `UiTriggers` trigger at subscription, and I want to skip that, and instead trigger when `Height` and `Width` are changed, after layout pass has already occurred

Make sense?

Reply:
> the principle of least astonishment

Also known as the No WTFs rule 🤣

Reply:
> MHO, this isn't a good way to fix the problem. The real reason why it updates twice is that UiTriggers include the usage of Throttle. Skipping the first 2 notifications is brittle.

This is just simply false.
It updates twice because there are two triggers for it. The first one is `PrivacyProgressUpdateTrigger` and the second one is the `Width` and `Height`.
And not because there is a throttle.

> Nobody would expect to get a "PrivacyProgressUpdate" notification 1.1 seconds after having subscribed to it, even when there's no update at all. This can make sense for some scenarios.

I discussed this with @SuperJMN on Slack, and he also agreed this is the best we can do. Although taking another look at the `UiTriggers` we cannot remove the throttle from everywhere (i.e. AnonScore), but for this case `Throttle` is not the best solution as we just want to avoid the double trigger on subscription.

So just simply remove the throttle from `UiTriggers`, and add a `Skip(1)`, so it will only trigger once, which we want.

Regarding the issue that this PR fixes, You have to merge the two triggers to have only one. Once you replaced the throttle with the skip in the UiTriggers, it should be an easy task.

Reply:
I took a look at this UiTriggers and the throttle cannot be removed from the `AnonScoreTargetChanged` as even if it changes in the CoinjoinSetting, it won't be stored in the KeyManager, and all of the subscribers take the value from the `KeyManager`.

This is a workaround because we don't want NotifyPropertyChange in the business code.

Reply:
@ichthus1604 @SuperJMN It seems the anonscore cannot be changed anymore on the Coinjoin settings page, thus I was able to remove some code and also remove the throttles from `UiTriggers`.
Please take a look https://github.com/zkSNACKs/WalletWasabi/pull/9373